app-devel/binutils+cross-ppc64el/autobuild/build

```diff
@@ -0,0 +1,21 @@
+unset CFLAGS CXXFLAGS CPPFLAGS LDFLAGS
```

Comment:
You might want to copy the scripts from `mips64r6el`, which has been linted before.